clients/mssql/dialect/dialect.go

```diff
@@ -44,13 +44,9 @@ func (MSSQLDialect) IsTableDoesNotExistErr(err error) bool {
 }
 
 func (md MSSQLDialect) BuildIsNotToastValueExpression(tableAlias constants.TableAlias, column columns.Column) string {
+	toastedValue := "%" + constants.ToastUnavailableValuePlaceholder + "%"
 	colName := sql.QuoteTableAliasColumn(tableAlias, column, md)
-	// Microsoft SQL Server doesn't allow boolean expressions to be in the COALESCE statement.
-	if column.KindDetails == typing.Struct {
-		return fmt.Sprintf("COALESCE(%s, {}) != {'key': '%s'}", colName, constants.ToastUnavailableValuePlaceholder)
-	}
-
-	return fmt.Sprintf("COALESCE(%s, '') != '%s'", colName, constants.ToastUnavailableValuePlaceholder)
+	return fmt.Sprintf("COALESCE(%s, '') NOT LIKE '%s'", colName, toastedValue)
```

Comment:
Could you just do
```
return fmt.Sprintf("COALESCE(%s, '') NOT LIKE '%%s%'", colName, constants.ToastUnavailableValuePlaceholder)
```
instead of defining `toastedValue` above, or do the extra `%`s mess with `fmt.Sprintf`?

Reply:
(I guess that's not really more readable though, so I'm also good with it as-is)

Reply:
Ah I did it like this because `%` is used to start string interpolation

![image](https://github.com/user-attachments/assets/860cdfd4-1aee-46c0-8bae-ef0569892e92)

Reply:
Ah nice, that's what I suspected